generics/services/report.js

```diff
@@ -7,151 +7,138 @@
 
 //dependencies
 
-const request = require('request');
-const reportsUrl = process.env.ML_REPORTS_SERVICE_URL;
+const request = require('request')
+const reportsUrl = process.env.ML_REPORTS_SERVICE_URL
 
 /**
```

Comment:
@MallanagoudaB do we need this file?

Reply:
@MallanagoudaB we don't need this file. we now don't have any dependencies with report service
Why you resolved this without actually resolving it?